.storybook/pages/AlongDemo/GlobalStyles.module.css

```diff
@@ -62,75 +62,71 @@
   --along-color-sky-500: #d8f6fd;
   --along-color-eraser-red-200: #cd2026;
 
-  /* Level 1 EDS tokens */
+  /* Override EDS tokens to apply the Along theme */
   --eds-border-radius-md: 0.5rem;
-
-  /* Level 2 tokens */
   --eds-font-family-primary: 'Untitled Sans', sans-serif;
   --eds-font-family-secondary: 'Moranga', serif;
-  --eds-theme-color-focus-ring: var(--along-color-blurple-500);
-  --eds-theme-color-text-neutral-default: var(--along-color-space-500);
+  --eds-theme-color-background-brand-primary-default: var(--along-color-white);
   --eds-theme-color-background-disabled: var(--along-color-space-100);
-  --eds-theme-color-border-disabled: var(--along-color-space-100);
-  --eds-theme-color-icon-disabled: var(--along-color-space-100);
-  --eds-theme-color-text-disabled: var(--along-color-space-100);
+  --eds-theme-color-background-neutral-medium: var(--along-color-sneaker-100);
   --eds-theme-color-background-neutral-subtle: var(--along-color-sneaker-100);
+  --eds-theme-color-border-brand-primary-strong: var(--along-color-blurple-500);
+  --eds-theme-color-border-disabled: var(--along-color-space-100);
   --eds-theme-color-border-link-brand: var(--along-color-space-500);
   --eds-theme-color-border-link-neutral: var(--along-color-space-500);
-  --eds-theme-color-icon-link-default: var(--along-color-space-500);
-  --eds-theme-color-icon-link-default-hover: var(--along-color-space-500);
-  --eds-theme-color-text-link-brand: var(--along-color-space-500);
-  --eds-theme-color-text-link-neutral: var(--along-color-space-500);
-  --eds-theme-color-background-brand-primary-default: var(--along-color-white);
-  --eds-theme-color-border-brand-primary-strong: var(--along-color-blurple-500);
-  --eds-theme-color-form-border: var(--along-color-sneaker-300);
-  --eds-theme-color-icon-brand-primary: var(--along-color-blurple-500);
-  --eds-theme-color-background-neutral-medium: var(--along-color-sneaker-100);
   --eds-theme-color-border-neutral-default: var(--along-color-space-200);
-
-  /* Level 3 tokens */
```

Comment:
The removal of these 2 lines is the only change in the EDS variable override section aside from sorting.